### PR TITLE
automatical redirect from index page, logging, nimor improvements

### PR DESCRIPTION
**Pull request by kvazar412**
added automatical redirect for index page, added logging for some classes - pay attention to log-file path, minor improvements

### CODE REVIEW

File: StingerInfoUI/pom.xml [hunk 1]
```diff
@@ -10,12 +10,14 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <hibernate.version>5.2.3.Final</hibernate.version>
         <jdk.version>1.8</jdk.version>
-        <mysql.connector.version>6.0.5</mysql.connector.version>
+        <mysql-connector.version>5.1.39</mysql-connector.version>
```
Comment (StShadow) [inline StingerInfoUI/pom.xml]: Why version downgrade?

File: StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java [hunk 2]
```diff
@@ -25,16 +28,25 @@ public void setDbService(DbService dbService) {
     // ******* Mapping urls of searchPage.jsp *******
     @RequestMapping(value = "searchPage")
     public String searchPage() {
+        if(LOGGER.isInfoEnabled()){
```
Comment (StShadow) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: with slf4j you don't need to do isXXXEnable() checks, the check will be done by slf4 itself - that's why this framework is so cool.

File: StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java [hunk 3]
```diff
@@ -25,16 +28,25 @@ public void setDbService(DbService dbService) {
...
+            LOGGER.info("Processing searchPage url"); 
+        }
         return "searchPage";
     }
 
     // ******* Mapping urls of movieList.jsp *******
     @RequestMapping(value = "/createOrUpdateMovie", method = RequestMethod.POST)
-    public String createMovie(@Valid Movie movie, BindingResult bindingResult) {
+    public String createMovie(@Valid Movie movie, BindingResult bindingResult) {
```
Comment (StShadow) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: extrawhite-space at the end of line

File: StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java [hunk 4]
```diff
@@ -25,16 +28,25 @@ public void setDbService(DbService dbService) {
...
         return "searchPage";
     }
 
     // ******* Mapping urls of movieList.jsp *******
     @RequestMapping(value = "/createOrUpdateMovie", method = RequestMethod.POST)
-    public String createMovie(@Valid Movie movie, BindingResult bindingResult) {
+    public String createMovie(@Valid Movie movie, BindingResult bindingResult) {        
         if (bindingResult.hasErrors()) {
+            if(LOGGER.isInfoEnabled()){
+                LOGGER.info("Movie input form has {} error(s)", bindingResult.getErrorCount());
```
Comment (StShadow) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: if there are errors - it's not info anymore :) level Error or at least Warning.

File: StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java [hunk 5]
```diff
@@ -43,32 +55,44 @@ public String createMovie(@Valid Movie movie, BindingResult bindingResult) {
...
 
     @RequestMapping(value = "/movieList", method = RequestMethod.GET)
     public String movieList(Model model) {
-        // VoteId voteId1 = new VoteId("123456", "123456");
-        // Vote vote1 = new Vote(voteId1, dbService.readMovieById("123456"),
-        // dbService.readUserById("123456"), VoteValue.YES);
         model.addAttribute("movie", new Movie());
         model.addAttribute("movieList", this.dbService.readAllMovies());
+        if(LOGGER.isInfoEnabled()){
+            LOGGER.info("Processing movieList url");
```
Comment (StShadow) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: Usually such pure debug info is logged with debug (or even trace) levels - a log full of noise is very hard to read.

Comment (kvazar412) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: ok, done

File: StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java (hunk 4 above)
Comment (kvazar412) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: agree with you. changed to "warn"

File: StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java (hunk 3 above)
Comment (kvazar412) [inline StingerInfoUI/src/main/java/org/dtmhapcs/controller/CrudController.java]: done

**Pull request merged**